python/ucxx/_lib/libucxx.pyx

```diff
@@ -420,22 +403,31 @@ cdef class UCXAddress():
 
     # For old UCX-Py API compatibility
     @classmethod
-    def from_worker(cls, UCXWorker worker):
+    def from_worker(cls, UCXWorker worker) -> UCXAddress:
+        cdef UCXAddress address = UCXAddress.__new__(UCXAddress)
+
+        warnings.warn(
+            "UCXAddress.from_worker() is deprecated and will soon be removed, "
+            "use UCXWorker.create_from_worker() instead",
```

Comment:
```suggestion
            "use UCXAddress.create_from_worker() instead",
```

Reply:
Good catch, the warning is wrong and your suggestion is what it should read. Accessing through `UCXWorker.address` is merely a convenience and they're 100% equivalent, in fact `UCXWorker.address` calls `UCXAddress.create_from_worker()`, as you've probably seen already.